app/src/main/java/com/nitramite/paketinseuranta/MainMenu.java

```diff
@@ -227,19 +232,6 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
 
-        // Install security provider, which is not supported by android kitkat for ssl connections
-        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.KITKAT) {
-            try {
-                ProviderInstaller.installIfNeeded(getApplicationContext());
-            } catch (GooglePlayServicesRepairableException e) {
-                e.printStackTrace();
-            } catch (GooglePlayServicesNotAvailableException e) {
-                e.printStackTrace();
-            }
-        }
```

Comment:
Should this have been removed? I can't find a related commit.

Reply:
Not in this branch for sure. 😭

Reply:
I guess you're not planning to revert it from this branch so I'll just approve. :slightly_smiling_face: